packages/help-center/src/components/help-center-support-chat-message.tsx

```diff
@@ -32,10 +32,12 @@ export const HelpCenterSupportChatMessage = ( {
 	message,
 	sectionName,
 	conversation,
+	unreadMessages = 0,
 }: {
 	message: OdieMessage | ZendeskMessage;
 	sectionName?: string;
 	conversation: OdieConversation | ZendeskConversation;
+	unreadMessages?: number;
```

Comment:
I would be more explicit here as it's not always clear looking at the code below what that variable contains without checking its type (e.g. one could expect a list of unread messages):
```suggestion
	numberOfUnreadMessages?: number;
```